ana_scans.py

```diff
@@ -163,7 +163,7 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
   from anaoptions import parser
 
   parser.add_option("--anaType", type="string", dest="anaType",
-                    help="Analysis type to be executed, from list {'latency','scurve','threshold','trim'}", metavar="anaType")
+                    help="Analysis type to be executed, from list {'latency','scurve','thresholdch','thresholdvftrig','thresholdvftrk','trim'}", metavar="anaType")
```

Comment:
Actually instead of having the list printed manually can we just use string concatenation & list comprehension here to insert `ana_config.keys()`? This would ensure a copy/paste error is not committed and be easier to maintain.